### DISCUSSION TITLE
Signing for UnrarKit-UnrarKitResources

### DISCUSSION THREAD

**Discussion by gpotari** [abbeycode/UnrarKit#104] [General]
Hi @abbeycode, 
May I ask why do we need to force code signing for the unrarkit resources? 
Every time we run pod install, we need to set it again. (See screenshot attached) 
<img width="1313" alt="Screenshot 2022-07-01 at 12 14 50" src="https://user-images.githubusercontent.com/17654144/176884203-5ef7e740-aaa7-4b30-9baa-98b56469dc8a.png">

Does it solve this issue if we uncheck Automatically manage signing for the Resources target? 

<img width="1174" alt="Screenshot 2022-07-01 at 12 16 31" src="https://user-images.githubusercontent.com/17654144/176884411-e07e7682-8b5b-4d79-94dd-1e043ea8dd83.png">

Thank you In advance

**Comment by gpotari** [comment]
@abbeycode Could you elaborate on this please? Thank you

**Comment by abbeycode** [comment]
That's a great thought. That wasn't a setting I ever intentionally turned on. Can you try turning it off and submit a PR to make sure the automated checks pass? I don't think changing it in the Xcode project will help for when it's installed with CocoaPods though, since it doesn't use the checked-in Xcode project directly.

Thanks for all the attention you've been giving UnrarKit!

**Comment by gpotari** [comment]
Sorry @abbeycode , but looks like my latest PR didn't solve the code signing problem we had. 

Can I ask what is the propose of having an extra Resources target for the solution ? I can see that single localisation string file we have. Can we add it inside the main project and remove Resources target?  
Thank you

**Comment by sjmills** [comment]
This is causing a build error for me. I made some changes to a project I haven't worked on in a while, using the latest Xcode, and when I try to build an archive for the App Store, I get 2 errors:

UnrarKit:
UnrarKit/Classes/URKFileInfo.m:66:9 Mixing declarations and code is incompatible with standards before C99
UnrarKitResources:
Ad Hoc code signing is not allowed with SDK 'iOS 16.4'

I've switched to the latest branch 2.11. Same thing.
Same problems with UnzipKit.
I'd like to release a new build of my app, but these errors are preventing that.
It also prevents me from targeting an actual device, so I have to test on the Simulator, which is not always ideal.
